### PR TITLE
Improve safe to test label usage in manager ci

### PR DESCRIPTION
**Pull request by r00ta**
[Insert JIRA Issue number and update link](https://issues.redhat.com/browse/MGDOBR-)

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.

</details>

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]

File: .github/workflows/Manager-CI.yaml [hunk 1]
```diff
@@ -26,14 +25,19 @@ jobs:
...
     steps:
       - name: Checkout
         uses: actions/checkout@v2
         with:
           ref: "refs/pull/${{ github.event.number }}/merge"
+      - name: Check labels
+        # Security due to https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
+        if: ${{ !contains( github.event.pull_request.labels.*.name, 'safe to test') }}
+        run: |
+          echo "Please add the 'safe to test' label in order to run this pipeline if it's safe to test this code"
```
Comment (jomarko) [inline .github/workflows/Manager-CI.yaml:39]:
```suggestion
          echo "Please add the 'safe to test' label in order to run 'Manager - CI' pipeline if it's safe to test this code"
```

**Review by jomarko — COMMENTED** [review]
Thank you @r00ta , just one comment/proposal. Please let me know what do you think.

**Comment by r00ta** [comment]
> Thank you @r00ta , just one comment/proposal. Please let me know what do you think.

Done

**Review by jomarko — APPROVED** [review]
Thank you @r00ta

**Pull request merged**